tests/test_adjoint_jacobian.py

```diff
@@ -320,12 +320,9 @@ def test_gradients(self, op, obs, tol, dev, C):
             qml.expval(obs(wires=0))
             qml.expval(qml.PauliZ(wires=1))
 
-        tape.execute(dev)
+        dev.trainable_params = set(range(1, 1 + op.num_params))
```

Comment:
@maliasadi can this line be deleted?

Reply:
This is added to the `grad_F` lambda expr.